db/seeds.rb

```diff
@@ -613,7 +613,7 @@
       exists = true if s.section.to_s == details[:section] && s.section.organisation == details[:organisation]
```

Comment:
With how current the data is seeded (checking only the text, and section and their organization), this text change in the guidance will not be picked up. We'd need to change the code here to update guidance for existing questions, or for this fix, change the data in the database directly?